src/ErrorHandler.gs.js

```diff
@@ -629,7 +664,23 @@ ErrorHandler_._ERROR_PARTIAL_MATCH = [
     variables: ['timestamp'],
     ref: NORMALIZED_ERRORS.USER_RATE_LIMIT_EXCEEDED_RETRY_AFTER_SPECIFIED_TIME,
     locale: 'en'},
-
+  
+  // "Invalid argument: XXX" - wrong email alias used - eg: GmailApp.sendEmail()
+  {regex: /^Invalid argument: (.*)$/,
```

Comment:
If this error is specific to an Email sending method, then okay to keep the 'email' variable.
If not, then maybe a more generic name will be better